package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@fnndsc/chrisapi": "0.0.4",
+    "@fnndsc/chrisapi": "0.0.5",
```

Comment:
would you mind bumping this to 0.0.6 and giving a quick test to make sure everything still works :)

Reply:
Bumped and everything seems fine. I think the bug fix might let us remove the extra `client.getFeeds()` calls - I'll look into that.